typescript/sdk/src/token/altVMDeploy.ts

```diff
@@ -38,9 +44,7 @@ export class CosmosNativeDeployer {
 
     const configMapToDeploy = objFilter(
```

Comment:
This variable reverts to type `any` and so further errors are hidden. You can delcare it explicitly with
```
type ConfigMap = WarpRouteDeployConfigMailboxRequired;
const configMapToDeploy: ConfigMap = objFilter(...);
```